app/controllers/users_controller.rb

```diff
@@ -24,6 +24,12 @@ def create
     end
   end
 
+  def edit
+    user_info = request.env['omniauth.auth']
+    current_user.update(github_token: user_info["credentials"]["token"])
+    redirect_to dashboard_path
+  end
```

Comment:
This makes sense to edit the user here as opposed to updating the Sessions Controller.